samples/scripts/deployment/sshDeploymentOneJar

```diff
@@ -9,6 +9,7 @@ HOST_NAME=$1
 NODE_NAME=$2
 NODE_SOURCE_NAME=$3
 RM_URL=$4
+NB_NODES=$5
```

Comment:
This change could break existing scripts that use _sshDeploymentOneJar_. To prevent that, you could check if $5 is defined, if not then a default value should be set.